kvdb-rocksdb/src/lib.rs

```diff
@@ -112,11 +112,7 @@ impl CompactionProfile {
 		let hdd_check_file = db_path
 			.to_str()
 			.and_then(|path_str| Command::new("df").arg(path_str).output().ok())
-			.and_then(|df_res| if df_res.status.success() {
-				Some(df_res.stdout)
-			} else {
-				None
-			})
+			.and_then(|df_res| if df_res.status.success() { Some(df_res.stdout) } else { None })
```

Comment:
Is this a rustfmt change? I don't mind it, just curious.

Reply:
right, you can see it in the commit `cargo fmt`, otherwise ci was failing https://travis-ci.org/paritytech/parity-common/jobs/617128841#L217-L248

Reply:
(rustfmt was enforced in #266)